src/pages/calculator.js

```diff
@@ -1,350 +1,346 @@
-                style={{
-                  display: this.state.showCpiDropdown ? "block" : "none"
-                }}
-                className="dropdown-menu"
-                aria-labelledby="dropdownMenuButton"
-              >
-                <a
-                  onClick={() => {
-                    this.setState({
-                      showCpiDropdown: !this.state.showCpiDropdown,
-                      cpiSelection: "Oakland-Hayward-San Francisco",
-                      cpi: 0.04
-                    });
-                  }}
-                  className="dropdown-item"
-                >
-                  Oakland-Hayward-San Francisco
-                </a>
-                <a
-                  onClick={() => {
-                    this.setState({
-                      showCpiDropdown: !this.state.showCpiDropdown,
-                      cpiSelection: "Los Angeles-Long Beach-Anaheim",
-                      cpi: 0.033
-                    });
-                  }}
-                  className="dropdown-item"
-                >
-                  Los Angeles-Long Beach-Anaheim
-                </a>
-                <a
-                  onClick={() => {
-                    this.setState({
-                      showCpiDropdown: !this.state.showCpiDropdown,
-                      cpiSelection: "San Diego-Carlsbad",
-                      cpi: 0.022
-                    });
-                  }}
-                  className="dropdown-item"
-                >
-                  San Diego-Carlsbad
-                </a>
-                <a
-                  onClick={() => {
-                    this.setState({
-                      showCpiDropdown: !this.state.showCpiDropdown,
-                      cpiSelection: "Riverside-San Bernardino-Ontario",
-                      cpi: 0.028
-                    });
-                  }}
-                  className="dropdown-item"
-                >
-                  Riverside-San Bernardino-Ontario
-                </a>
-                <a
-                  onClick={() => {
-                    this.setState({
-                      showCpiDropdown: !this.state.showCpiDropdown,
-                      cpiSelection: "Other",
-                      cpi: 0.033
-                    });
-                    document.querySelector("#continue").scrollIntoView();
-                  }}
-                  className="dropdown-item"
-                >
-                  Other
-                </a>
+            <h5 className="card-title">What is your zip code?</h5>
+            <input className="form-control" type="text" onChange={(e) => this.setCpiFromZip(e)} placeholder="Your 5 digit zip code" />
+            {this.state.town
+              && (
+              <small><strong>{this.state.town}</strong>{this.state.county
+                && <strong>, {this.state.county} County</strong>}
+              </small>
+              )}
+            <br />
+            <br />
+            <br />
```

Comment:
I mean, it's not like this is invalid but...

Reply:
yeah this is hacky, should be done w/ margin IMO

Reply:
This should totally be <p></p><p></p><p></p> like in my code